### PR TITLE
update page mode regarding mc 2016 launch

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by tanyaparker** [comment]
I'm not sure I understand if we need `page_mode = PageMode.get(request)` or not. The dashboard views seemed to work without it before.

File: pegasus/sites.v3/code.org/public/learn/index.haml [hunk 1]
```diff
@@ -81,8 +79,6 @@ social:
   - tutorial[:string_platforms]                     = hoc_s(prefix + "string_platforms")
   - tutorial[:string_detail_platforms]              = hoc_s(prefix + "string_detail_platforms")
   - tutorial[:string_detail_programming_languages]  = hoc_s(prefix + "string_detail_programming_languages")
-  - if tutorial[:short_code] == "mchoc" && page_mode == "feature-minecraft2016"
-    - tutorial[:image] = "/images/tutorials/new/minecraft2016.png"
```
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/learn/index.haml:85]: I updated the image in gsheet.

File: pegasus/sites.v3/code.org/public/minecraft.haml [hunk 2]
```diff
@@ -19,101 +19,59 @@ social:
...
+  =I18n.t(:minecraft_no_internet)
+  %a{href: CDO.studio_url("/download/mc")}= I18n.t(:minecraft_download_adventurer)
+
+.color-block.mc-store
+  %h2.white-text= I18n.t(:minecraft_store_camps)
+  = I18n.t(:minecraft_signup_store_camps).gsub('%{signup_store_camp}', 'https://www.microsoft.com/youthsparkprograms')
+
+.color-block.mc-host-event
+  %h2.white-text= I18n.t(:minecraft_host_event)
+  %a{href: '/hourofcode/mc'}= I18n.t(:minecraft_teach_resources)
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/minecraft.haml:60]: still the right link?

File: pegasus/sites.v3/code.org/public/minecraft.haml [hunk 3]
```diff
@@ -19,101 +19,59 @@ social:
...
+  %br
+  %a{href: 'https://docs.com/hourofcode2016'}= I18n.t(:minecraft_leader_guide)
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/minecraft.haml:62]: this one too?

**Comment by breville** [comment]
lgtm

**Comment by breville** [comment]
It looks like request is optional but is the way we can check for the cookie.  https://github.com/code-dot-org/code-dot-org/blob/staging/lib/dynamic_config/page_mode.rb#L22

File: pegasus/sites.v3/code.org/public/minecraft.haml (hunk 2 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/minecraft.haml:60]: Yes.

File: pegasus/sites.v3/code.org/public/minecraft.haml (hunk 3 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/minecraft.haml:62]: Yes.

**Pull request merged**